src/RelayKnownArgumentNames.ts

```diff
@@ -102,6 +103,20 @@ function validateFragmentArgumentDefinitions(context: ValidationContext, directi
   }
 }
 
+function isNullableArgument(argumentDefinition: ObjectValueNode): boolean {
+  const typeField = argumentDefinition.fields.find(f => f.name.value === "type")
+  if (typeField == null) {
+    return false
+  }
+
+  if (typeField.value.kind !== "StringValue") {
+    return false
+  }
+  const type = parseType(typeField.value.value)
```

Comment:
This needs a `try/catch` as it will throw a `SyntaxError` for unexpected tokens.